### PR TITLE
walkthrough: add links to debug tutorial

### PR DESCRIPTION
**Pull request by Eskibear**
image

linked to https://code.visualstudio.com/docs/java/java-debugging

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -126,7 +126,7 @@
           {
             "id": "java.showDebugView",
             "title": "Launch and debug your app",
-            "description": "The debugger will automatically find the entry point of your project, launch it and start debugging.\n[Show Run and Debug View](command:workbench.view.debug)",
+            "description": "The debugger will automatically find the entry point of your project, launch it and start debugging.\n[Show Run and Debug View](command:workbench.view.debug)\n[Learn More...](https://code.visualstudio.com/docs/java/java-debugging)",
```
Comment (jdneo) [inline package.json:129]: I'm thinking that if it would be helpful if trigger a command to open the link here?

Comment (Eskibear) [reply package.json:129]: what for? telemetry?

Comment (jdneo) [reply package.json:129]: yes

Comment (Eskibear) [reply package.json:129]: Pros:
- We'll tell how many users visit debugging docs from walkthrough (vs overview/welcome/etc.)

Cons:
- We'll have to activated the extension pack on executing that command.
- This link will look complicated and hard to maintain?

Comment (jdneo) [reply package.json:129]: > We'll have to activated the extension pack on executing that command.

Ok, this is a good point we should not use a command

**Review by jdneo — APPROVED** [review]

**Pull request merged**